modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+= Using skip node annotation
+
+* Use skip-node annotation to prevent the `node-labeller` from reconciling a node.
```

Comment:
This is the only place where "skip node" is hyphenated. Is that intentional? If so, I think it should be in backticks so that it's clear that you're referencing the annotation/command argument.